domains.txt

```diff
@@ -539,6 +597,7 @@ psdgroup.com
 psr-rec.com
 puredevelopers.co.uk
 purple-consultancy.com
+purviewservices.co.uk
```

Comment:
Purview Services was founded in 2010 with the mission to provide best-in-class IT consultancy services across the globe. We are a specialist IT consultancy offering services through our consultants/experts on a contingency basis in the UK, Europe.